Tests/SentryTests/Helper/TestNSNotificationCenterWrapper.swift

```diff
@@ -1,19 +1,27 @@
 import Foundation
 
-class TestNSNotificationCenterWrapper: SentryNSNotificationCenterWrapper {
+@objc public class TestNSNotificationCenterWrapper: SentryNSNotificationCenterWrapper {
     
     var addObserverInvocations = Invocations<(observer: Any, selector: Selector, name: NSNotification.Name)>()
-    override func addObserver(_ observer: Any, selector aSelector: Selector, name aName: NSNotification.Name) {
+    @objc public var addObserverInvocationsCount: Int {
+        return addObserverInvocations.count
+    }
+    
+    public override func addObserver(_ observer: Any, selector aSelector: Selector, name aName: NSNotification.Name) {
         addObserverInvocations.record((observer, aSelector, aName))
     }
     
     var addObserverWithNotificationInvocations = Invocations<(observer: Any, name: NSNotification.Name)>()
-    override func removeObserver(_ observer: Any, name aName: NSNotification.Name) {
+    @objc public var removeWithNotificationInvocationsCount: Int {
```

Comment:
I don't really understand the logic behind this. We have a `removeObserver` function override which adds to `addObserverWithNotificationInvocations` which seems very weird, and then we now added another property called `removeWithNotificationInvocationsCount` which is just the count of that wrongly named property.

In #2460 I just renamed `addObserverWithNotificationInvocations` to something more sensible and got rid of this other property, but I am trying to understand why it was done like this in the first place?

Reply:
I think this was a mistake on my side. It should be `removeObserverWithNotificationInvocations`

Reply:
Well it got fixed in #2460 anyway :)